firstvoices/backend/admin/characters_admin.py

```diff
@@ -79,6 +76,11 @@ class CharacterAdmin(BaseSiteContentAdmin):
     ) + BaseSiteContentAdmin.list_display
     search_fields = ("title", "approximate_form")
     inlines = (CharacterVariantInline, CharacterRelatedDictionaryEntryInline)
+    autocomplete_fields = ("related_audio", "related_images", "related_videos")
```

Comment:
Local testing result: the related media fields on characters are not filtered by site